code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -590,6 +590,64 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 /datum/ammo/bullet/shotgun/spread/masterkey
 	damage = 30
 
+/datum/ammo/bullet/shotgun/sx16_buckshot
+	name = "shotgun buckshot shell"
+	icon_state = "buckshot"
+	hud_state = "shotgun_buckshot"
+	bonus_projectiles_type = /datum/ammo/bullet/shotgun/sx16_buckshot/spread
+	bonus_projectiles_amount = 2
+	bonus_projectiles_scatter = 10
+	accuracy_var_low = 10
+	accuracy_var_high = 10
+	max_range = 10
+	damage = 40
+	damage_falloff = 2
+	penetration = 10
```

Comment:
Does buckshot need pen?

Reply:
i guess not, it barely has any so ill just remove it